src/miniscript/types/malleability.rs

```diff
@@ -138,6 +138,14 @@ impl Property for Malleability {
         }
     }
 
+    fn from_txtemplate() -> Self {
+        Malleability {
+            dissat: Dissat::None,
+            safe: true,
+            non_malleable: true,
```

Comment:
I don't if this should be regarded as safe. In Mini script, the definition of safe is "Signed: satisfying this expression always requires a signature". Another way of looking at this if CTV as the only top-level script is safe. 

I have not (yet) read CTV, so I am unsure about this.

Reply:
lifting this to the main thread